src/app/retrospective/retrospective-dashboard/retrospective-dashboard.component.ts

```diff
@@ -72,6 +75,23 @@ export class RetrospectiveDashboardComponent implements OnInit, OnDestroy {
         return (this.retrospectiveData.CreatedBy.FirstName + ' ' + this.retrospectiveData.CreatedBy.LastName).trim();
     }
 
+    showUpdateRetroDialog() {
+        const dialogRef = this.dialog.open(RetrospectiveCreateComponent, {
+            width: '90%',
+            height: '90%',
+            maxWidth: 950,
+            data : {
+                retrospective: this.retrospectiveData,
+            }
+        });
+
+        dialogRef.afterClosed().takeUntil(this.destroy$).subscribe(result => {
+            if (result) {
+                this.refreshSprintsList();
```

Comment:
Why do we need to refresh the sprint list?

Reply:
Removed.